File: week-05/day-05/garden-again/gardening.ts
import { Garden } from "./garden";
import { Flower } from "./flower";
import { Tree } from "./tree";

'use strict'

let myGarden: Garden = new Garden;
let yellowFlower: Flower= new Flower('yellow')
let blueFlower: Flower= new Flower('blue')
let purpleTree: Tree= new Tree('purple')
let orangeTree: Tree= new Tree('orange')
myGarden.addToGarden(yellowFlower)
myGarden.addToGarden(blueFlower)
myGarden.addToGarden(purpleTree)
myGarden.addToGarden(orangeTree)

myGarden.watering(40)
myGarden.watering(70)

File: datascience/week-01/garden/garden_py/gardening.py
import random
import sys
import os
from plant import Plants
from garden import Garden
from flower import Flower
from tree import Tree

myGarden = Garden()

yellowFlower = Flower('yellow')
blueFlower = Flower('blue')
purpleTree = Tree('purple')
orangeTree = Tree('orange')

myGarden.addToGarden(yellowFlower)
myGarden.addToGarden(blueFlower)
myGarden.addToGarden(purpleTree)
myGarden.addToGarden(orangeTree)

print('\n')
myGarden.printGarden()
print('\n')
myGarden.watering(20)
myGarden.printGarden()
print('\n')
myGarden.watering(30)
myGarden.printGarden()
print('\n')
myGarden.watering(40)
myGarden.printGarden()
